boards/nano_rp2040_connect/layout.ld

```diff
@@ -2,13 +2,13 @@ MEMORY
 {
   /* uncomment this to boot from RAM */
   /* reset (rx)  : ORIGIN = 0x20000000, LENGTH = 16K
-  rom (rx)  : ORIGIN = 0x20000100, LENGTH = 128K
+  rom (rx)  : ORIGIN = 0x20000100, LENGTH = 256K
```

Comment:
Wouldn't the address and length of the next section need to be changed as well?

Reply:
I think it already was in the right position before, 0x40000 should be 256K.

Reply:
I see, thanks